### PR TITLE
Added the answer for the question: 'How Web Servers Works'

### PR DESCRIPTION
**Pull request by alokVerma749**
Here I added the appropriate answer for the question.

### CODE REVIEW

**Comment by bregman-arie** [comment]
You focused on how browsing works (mainly from client side) rather than how the web server works. Maybe you can add more details on how web server does what it does, with any specifics you feel are relevant.

**Comment by alokVerma749** [comment]
I have updated my Pull Request . Please review this.

**Comment by alokVerma749** [comment]
yes,it is copied from my college notes.

**Comment by alokVerma749** [comment]
Specified the source

File: exercises/devops/README.md [hunk 1]
```diff
@@ -252,6 +252,33 @@ Reliability, when used in DevOps context, is the ability of a system to recover
 
 <details>
 <summary>How a web server works?</summary><br><b>
+	According to MDN Web Docs
```
Comment (bregman-arie) [inline exercises/devops/README.md:255]: Move it the beginning of the line + reference to site

File: exercises/devops/README.md [hunk 2]
```diff
@@ -252,6 +252,33 @@ Reliability, when used in DevOps context, is the ability of a system to recover
...
+	
+We can understand web servers using two view points,which is :
```
Comment (bregman-arie) [inline exercises/devops/README.md:257]: space after command and remove the one before the colon

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: exercises/devops/README.md (hunk 1 above)
Comment (alokVerma749) [reply exercises/devops/README.md]: ok i will do this

**Comment by alokVerma749** [comment]
resolved the given change requests

**Pull request merged**